TYPO3-Solr/ext-solr
# [TASK] Remove unused tx_solr_cache DB tables

# What this pr does

There are SQL CREATE TABLE definitions for tx_solr_cache and tx_solr_cache_tags.

These tables have been added back then when TYPO3 did not always create cache tables.

However, since TYPO3 v9, these tables have been renamed and are automatically created, so the SQL definition can be removed.

The SQL tables are not referenced anywhere in the current source code.

# How to test

Re-install the extension, drop the tables and see if everything still works.

Fixes: #3636
